src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -414,6 +412,12 @@ void runtime::cpu::CPU_ExternalFunction::compile()
     writer << "// Generated by the nGraph CPU backend\n";
     if (m_use_tbb)
     {
+        if (runtime::cpu::IsTracingEnabled() || m_emit_timing)
+        {
+            NGRAPH_WARN
```

Comment:
Libraries should not print things. Remove the warning or find some way to get this information to an app like nbench.

Reply:
This is only going to kick in for ngraph developers. Dont want to spend the time implementing the plumbing for this. Will make this an assert for now.